src/autoschedulers/adams2019/autotune_loop.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
Does this specifically require `bash` (vs just `sh`)? OSX machines now default to `zsh`.

Reply:
It definitely _looks_ like bash to me. And it's not a valid executable script without a shebang, anyway, so something has to go up there. macOS still has bash installed and will for a while.